File: python/Lab ปี1 ภาคต้น/week4/code/hw4.py
# ข้อ 4
book= {
    1: int(input("ราคาหนังสือเล่มที่ 1 >> ")),
    2: int(input("ราคาหนังสือเล่มที่ 2 >> ")),
    3: int(input("ราคาหนังสือเล่มที่ 3 >> "))
}

_min = min(*book.values())
total = sum(list(book.values()))
print(f"ราคาหนังสือทั้ง 3 เล่ม {total} บาท")
print(f"ราคาหนังสือ 2 เล่ม {total - _min} บาท")

File: python/Lab ปี1 ภาคต้น/week4/code/hw4.c
#include <stdio.h>

void main(){
    int book1 , book2, book3 , total , min; 

    printf("Enter your a book1 price >> ");
    scanf("%d" , &book1);
    printf("Enter your a book1 price >> ");
    scanf("%d" , &book2);
    printf("Enter your a book1 price >> ");
    scanf("%d" , &book3);

    total = book1 + book2 + book3;

    if(book1 < book2 && book1 < book3){
        min = book1;
    } else if(book2 < book1 && book2 < book3) {
        min = book2;
    } else {
        min = book3;
    }

    printf("total of price a three book: %d" , total);
    puts("");
    printf("total of price a three book: %d" , total - min);
}